src/bundles/copy_gc/index.ts

```diff
@@ -88,14 +64,15 @@ function generateMemory(): void {
     desc: 'Memory initially empty.',
     leftDesc: '',
     rightDesc: '',
+    scan: -1,
+    free: -1,
   };
 
   commandHeap.push(obj);

```

Comment:
By directly defining the `obj` inside the `commandHeap.push()` function, you eliminate the need to pollute the scope's name space and specify the type of `obj` to be `CommandHeapObject` in line 55. This is not absolutely necessary though, I think its completely up to you.